hadoop-project-dist/pom.xml

```diff
@@ -341,7 +340,6 @@
                     <argument>--openssllib=${openssl.lib}</argument>
                     <argument>--opensslbinbundle=${bundle.openssl.in.bin}</argument>
                     <argument>--openssllibbundle=${bundle.openssl}</argument>
-                    <argument>--snappybinbundle=${bundle.snappy.in.bin}</argument>
                     <argument>--snappylib=${snappy.lib}</argument>
                     <argument>--snappylibbundle=${bundle.snappy}</argument>
```

Comment:
what about the others snappylibs?

Reply:
we don't touch hadoop-mapreduce-client-nativetask that needs snappy lib still, per https://github.com/apache/hadoop/pull/2201#issuecomment-681687572

Reply:
does this mean we don't need the option in dev-support/bin/dist-copynativelibs for snappy?

Reply:
Hmm, I am not sure about this now. I think it is safer to revert this back, I am not sure if hadoop-mapreduce-client-nativetask uses this.

Reply:
Re-checked. hadoop-mapreduce-client-nativetask doesn't use `bundle.snappy.in.bin`. Only native-win profile of hadoop-common and native-win profile in hadoop-project use `bundle.snappy.in.bin`. They are added by HADOOP-9802 to support SnappyCodec on Windows. So looks like it is safe to remove.